Tasks/VsTestV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -22,6 +22,8 @@
   "loc.input.help.tcmTestRun": "Test run based selection is used when triggering automated test runs from the test hub. This option cannot be used for running tests in the CI/CD pipeline.",
   "loc.input.label.searchFolder": "Search folder",
   "loc.input.help.searchFolder": "Folder to search for the test assemblies.",
+  "loc.input.label.resultsFolder": "Test results folder",
+  "loc.input.help.resultsFolder": "Folder to store test results. When this input is not specified, results are stored in $(Agent.TempDirectory) by default, which is cleaned at the end of a pipeline run. When the default value is overridden, the specified folder will be cleaned by the task before tests are run.",
```

Comment:
The specified/default results directory will always be cleaned up at the start of the vstest task.

Reply:
Done